src/components/SearchBar.js

```diff
@@ -146,23 +146,22 @@ const SearchBar = () => {
           </ClearSearch>
         )}
       </SearchOutline>
-      <Animated
-        animationIn="fadeIn"
-        animationOut="fadeOut"
-        animationInDuration={50}
-        animationOutDuration={50}
-        isVisible={searchResults.length === 0 && searchTerm.length > 1}
-        animateOnMount={false}
-        className="no-results"
-      >
-        <NoResultsText
-          isVisible={searchResults.length === 0 && searchTerm.length > 1}
+      {searchResults.length === 0 && searchTerm.length > 1 && (
```

Comment:
This makes much more sense, I have no idea why I didn't do it this way to start.